media.css

```diff
@@ -0,0 +1,364 @@
+
+  .nav-b {
+    background-color: #d3d3d3;
+    display: block;
+  }
+
+  .nav-b .menuList {
+    display: flex;
+    gap: 40px;
+    margin-left: 520px;
+    font-size: 15px;
+    margin-top: -70px;
+  }
+
+  .campaign {
+    margin-left: 35px;
+  }
+
+  .campaign:hover {
+    border: 3px solid #ec5342;
+    width: 110px;
+    text-align: center;
+    height: 50px;
+  }
+
+  .campaign a:hover {
+    color: #ec5342;
+  }
+
+  .nav-b .logo {
+    display: block;
+    margin-bottom: -55px;
+    width: 90px;
+    height: 90px;
+    margin-left: 200px;
+  }
+
+  .title3 {
+    font-size: 40px;
+    margin-bottom: 20px;
+    margin-top: 50px;
+    margin-left: 200px;
+    padding-top: 160px;
+  }
+
+  .title {
+    font-family: 'Cocogoose Pro', sans-serif;
+    color: #ec5342;
+    font-size: 49px;
+    width: 600px;
+    margin-bottom: 40px;
+    margin-left: 180px;
+  }
+
+  .intro {
+    background-size: cover;
+    margin-top: -50px;
+    height: 600px;
+  }
+
+  .intro::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-image: url(./img/background1.avif);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .intro1 {
+    font-size: 15px;
+    border: 1px solid #fff;
+    width: 610px;
+    margin-left: 190px;
+  }
+
+  .intro-date {
+    font-size: 25px;
+    margin-left: 190px;
+  }
+
+  .intro-location {
+    font-size: 20px;
+    margin-left: 190px;
+    margin-top: 15px;
+  }
+
+  .title1 {
+    font-size: 25px;
+  }
+
+  .hr1 {
+    width: 50px;
+  }
+
+  .info-list {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    grid-template-rows: repeat(1, 1fr);
+    margin: 0 auto;
+    margin-top: 60px;
+  }
+
+  .info-list li {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 14px;
+    width: 200px;
+    background-color: #ffffff1a;
+    list-style: none;
+    height: 250px;
+  }
+
+  .info-list li i {
+    font-size: 25px;
+  }
+
+  .info {
+    margin-top: 10px;
+    margin-left: 0;
+    width: 190px;
+    font-size: 19px;
+  }
+
+  .list1 {
+    text-align: center;
+    font-size: 16px;
+  }
+
+  .btn1 {
+    display: none;
+  }
+
+  .btn2 {
+    position: relative;
+    display: block;
+    width: 300px;
+    height: 40px;
+    margin: 0 auto;
+    margin-top: 50px;
+    color: #fff;
+  }
+
+  .btn2:hover {
+    font-size: 20px;
+  }
+
+  .hr2 {
+    margin-left: 600px;
+    width: 80px;
+    color: #ec5342;
+  }
+
+  .load-more {
+    display: none;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  .partner1 {
+    display: block;
+    border: 1px solid #272a31;
+    background-color: #272a31;
+  }
+
+  .partners2 {
+    margin-top: 10px;
+    font-family: 'Lato', sans-serif;
+    font-size: 15px;
+    text-align: center;
+    color: #d3d3d3;
+  }
+
+  .hr6 {
+    margin-top: 10px;
+    width: 30px;
+    margin-left: 625px;
```

Comment:
- [ ] Great Job giving this line the appropriate color as requested by the previous reviewer, however it is still not aligned to the center. To fix it you can use `margin: 10px auto 40px;` 

Original Desktop Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181594781-9826b292-2e12-4919-bc96-433f73d041cd.png)  |  ![Screenshot from 2022-07-29 15-44-27](https://user-images.githubusercontent.com/87965534/181763340-1d2883b0-367f-436e-8669-ea3e8f500ac8.png)